examples/feature-overview/amplitude.node.yml

```diff
@@ -4,3 +4,70 @@ settings:
   sdk: "@amplitude/amplitude-node"
   output: ./amplitude
   outputFileName: node
+  typedAnchorName: typed
+
+environments:
+  development:
+    experiment:
+      apiKey: my-deployment-key
+    analytics:
+      apiKey: my-api-key
+
+  production:
+    experiment:
+      apiKey: my-deployment-key
+    analytics:
+      apiKey: my-api-key
+
+x-analytics-definitions:
+  referralSource: &referralSource
+    description: How the user was brought to the app
+    enum: [ "facebook", "twitter", "other" ]
+
+x-experiment-definitions:
+  payload: &arrayOfStringsPayload
+    description: An array of strings
+    type: array
+    items:
+      type: string
+
+user:
+  description: Collection of user properties
+  required: [referralSource]
+  properties:
+    referralSource: *referralSource
+    favoriteSongCount:
+      description: Total number of favorited songs
+      type: integer
+
+analytics:
+  User Signed Up:
+    description: A user signed up
+    properties:
+      referralSource: *referralSource
+
+  User Logged In:
+    description: The user logged in
+    required: [ method ]
+    properties:
+      method:
+        description: The song unique identifier
+        enum: [ "email", "facebook", "google" ]
+
+  Add To Cart:
+    description: An item was added to the cart
+
+  Checkout:
+    description: The user completed their purchase
+
+experiments:
```

Comment:
Thanks @bgiori. This makes sense based on our call yesterday, but I was hoping to make the top-level entries in the YML to match the product names e.g. user, analytics, experiment, etc. I feel like "flags" at the top level would break from this pattern.

One options could be to nest "flags" under experiment, which could also make it easier to add more non-flag configuration in the future:

```
analytics:
   events:
       My Cool Event

experiment:
   flags:
       My Awesome Flag
```

WDYT?

Reply:
Updated to `analytics.events` and `experiment.flags`